src/team-service/teamRoutes.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import { generateTeam } from './teamController';
+import { authMiddleware } from '../libs/common/middleware/auth';
+
+const router = Router();
+
+router.use(authMiddleware);
+
+router.post('/generate', generateTeam);
```

Comment:
What is this API?!

Reply:
this api isnt necessary but i added it cuz i had all the logic rdy to use in other parts. an implementation of this could be to randomly generate a team for the user